lib/srv/reexec.go

```diff
@@ -619,10 +619,13 @@ func RunNetworking() (errw io.Writer, code int, err error) {
 	}
 
 	// Create a minimal default environment for the user.
-	homeDir := localUser.HomeDir
-	if !utils.IsDir(homeDir) {
-		homeDir = "/"
+	workingDir := string(os.PathSeparator)
```

Comment:
I think using `os.PathSeparator` here is misleading. I'd rather use plain "/" or define a constant:

```
const rootDirectory = "/"

...
	workingDir := rootDirectory
```

This should handle non-Windows platforms.